react/src/Components/HotelPage.jsx

```diff
@@ -172,57 +187,39 @@ const Hotel = () => {
+                {typeof value === 'object' ? (
+                  <div className="room-info">
+                    <div className="room-index"></div>
+                    <div className="star-rating">
+                      {[...Array(1)].map((_, index) => (
+                        <i key={index} className="fas fa-star"></i>
+                      ))}
+                    </div>
+                    {hotelImage && (
+                      <div className="custom-image-room">
+                        <img src={hotelImage} alt="Hotel" className="custom-image-room" />
+                      </div>
+                    )}
+                    <div className="room-details">
+                      {Object.entries(value).map(([subKey, subValue]) => (
+                        <div className="room-detail" key={subKey}>
+                          {subValue.toString()}
+                          <br /> {/* Add line break after each value */}
+                        </div>
+                      ))}
+                    </div>
+                  </div>
+                ) : (
+                  value.toString()
+                )}
               </div>
             ))}
-          </div>
-        ) : (
-          value.toString()
-        )}
-      </div>
-    ))}
-
-    {/* {Object.entries(room).map(([key, value]) => (
-      <div className="hotel_page_input3 room-container" key={key}>
-        
-
-      </div>
-    ))} */}
-    {/* <div className="hotel_page_input3 room-container" key={index}>
-      {Object.entries(room).map(([key, value]) => (
-        <div className="right-aligned" key={key}>
-          {value.type}
-        </div>
-      ))}
-    </div> */}
-
-
 
-            {/* <div className="hotel_page_input3 room-container">
-              <div className="right-aligned">{hotelRegulations}</div>
-              <div className="right-aligned">{hotelRegulations}</div>
-              <div className="right-aligned">{hotelRegulations}</div>
-              <div className="right-aligned">{hotelRegulations}</div>
+            <div className='button-container'>
+            <button className="order-button" onClick={() => handleButtonClick()}>رزرو</button>
             </div>
-            <div className="hotel_page_input3 room-container">
-              <div className="right-aligned">{hotelRegulations}</div>
-              <div className="right-aligned">{hotelRegulations}</div>
-              <div className="right-aligned">{hotelRegulations}</div>
-              <div className="right-aligned">{hotelRegulations}</div>
-            </div>
-            <div className="hotel_page_input3 room-container">
-              <div className="right-aligned">{hotelRegulations}</div>
-              <div className="right-aligned">{hotelRegulations}</div>
-              <div className="right-aligned">{hotelRegulations}</div>
-              <div className="right-aligned">{hotelRegulations}</div>
-            </div> */}
-
           </div>
         </div>
     </div>
```

Comment:
There are some differences between the implementation and the expected wireframe e.g.  comments, number of rooms, and reservation option.

Reply:
The wireframe you mentioned (Wireframe the [7th](https://www.figma.com/design/QTN5G7TzlPVf05ZtwQZJbi/Hotel-Reservation-System?node-id=13-301)) includes options for reservation and a comments section. These relate to other requirements such as the hotel reservation requirement and the comments section. The current version of the HotelPage.jsx addresses requirements 3.2.1 and 3.2.2 in the [SRS](https://docs.google.com/document/d/1WNsxSFMPqqLnsXX302NOK-ER4ilQDGUpj60miMzvgxw/edit?usp=sharing) document.